scripts/python3/install-dependencies.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -
```

Comment:
This logic is duplicated in the `test-venv.sh` file. Find out if there is a way to export functions in bash and make this an exportable function for re-use. If it's too tricky, leave it. The only thing we'll need is to run a bash file that looks like `test-venv.sh` file, where the arguments are passed in via a node `child_process`.